src/main/resources/resources/languages/languages_hu_HU.properties

```diff
@@ -1,6 +1,7 @@
 menuBar.fileMenu=Fájl
 menuBar.editMenu=Szerkesztés
 menuBar.toolsMenu=Eszközök
+menuBar.settingsMenu=Settings
```

Comment:
@gaborbata can you also please help to translate these to Hungarian language?

Reply:
Sure, it is called `Beállítások` in Hungarian